fused-effects.cabal

```diff
@@ -19,12 +19,15 @@ extra-source-files:
 tested-with:         GHC == 8.2.2
```

Comment:
Drop this line as well?

Reply:
Yeah, if we _do_ drop 8.2, this will definitely have to go with it.